sites/browserpod/src/content/docs/11-reference/00-BrowserPod/01-run.md

```diff
@@ -0,0 +1,32 @@
+---
+title: run
+description: Run a program inside the Pod
+---
+
+```ts
+class BrowserPod {
+	async run(
+		executable: string,
+		args: Array<string>,
+		opts?: { env?: Array<string>; cwd?: string }
+	): Promise<Process>;
+}
+```
+
+## Parameters
+
+- **executable (`string`)** - The path of the program to execute
+
+* **args (`Array<string>`, _optional_)** - The command line arguments to pass to the program.
+* **opts (`object`, _optional_)** - Initialization options passed as an object
+
+## Options
+
+- **env (`Array<string>`, _optional_)** - The environment variables to pass to the program, in the form "KEY=value".
```

Comment:
Please double check the expected format here, I think that maybe it's now `[key1, value1, key2, value2]`. It should be `"key=value"`